### ISSUE TITLE
(A) 'Skip to main content' button missing from data backend

### ISSUE THREAD

**Issue by otis-bath** [open-sdg/open-sdg#1754]
**High-level (A)** 
https://sdgdata.gov.uk/sdg-data/disaggregations.html
The DAC report highlighted that there is no skip link present to allow users reliant on the use of the keyboard alone to navigate to bypass blocks of content at the start of every web page and skip directly to the main page content. An example of this in action can be seen on the screenshot below:
![image](https://user-images.githubusercontent.com/90769108/182134335-df8543d2-37cc-4814-9dc7-180fe8b14499.png)

This is a persistent issue across the pages in the data backend. This should be on the pages with repetitive content and should be the first focusable control on the page, be descriptive that it links to the main content, activating the skip link moves the focus to the main content and that after activating the link, the keyboard focus has moved to the main content.
![image](https://user-images.githubusercontent.com/90769108/182134600-fb7452bf-b951-4c59-8294-e60283fb7ed1.png)
